nathandickinson32/second-nature
# Unit tests

Steps to test:
git pull
git checkout unit-tests
npm install
(ideally) npm run test

See if there's any errors and if you're able to run all of the tests in the only test file I have at the moment
